testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/nativeauth/EmailPasswordSignInSignUpFragment.kt

```diff
@@ -279,6 +283,8 @@ class EmailPasswordSignInSignUpFragment : Fragment() {
                 binding.resultAccessToken.text =
                     getString(R.string.result_access_token_text) + accessToken
 
+                Log.d("AccessToken", accessToken)
```

Comment:
Why is this needed? If not, let's not log this. This is sensitive data and we should be setting an example.